test/index.js

```diff
@@ -1,349 +1,241 @@
 'use strict'
+
 const assert = require('node:assert/strict')
+const crypto = require('node:crypto')
 const sinon = require('sinon')
 
 const Address = require('address-rfc2821')
 const fixtures = require('haraka-test-fixtures')
-const message = require('haraka-email-message')
-const { SPF } = require('haraka-plugin-spf')
 
 beforeEach(function () {
   this.plugin = new fixtures.plugin('bounce')
-  this.plugin.cfg = {
-    main: {},
-    check: {
-      single_recipient: true,
-      empty_return_path: true,
-      non_local_msgid: true,
-      bounce_spf: true,
-    },
-    reject: {
-      all_bounces: false,
-      single_recipient: true,
-      empty_return_path: true,
-      bad_rcpt: true,
-      non_local_msgid: true,
-      bounce_spf: true,
-    },
-    host_list: ['example.net'],
-  }
-
   this.connection = fixtures.connection.createConnection()
   this.connection.remote.ip = '8.8.8.8'
-  this.connection.transaction = {
-    header: new message.Header(),
-    results: new fixtures.results(this.plugin),
-    mail_from: new Address.Address('<>'),
-    rcpt_to: [new Address.Address('test@example.com')],
-  }
+  this.connection.relaying = false
+  this.connection.transaction = fixtures.transaction.createTransaction()
+  this.connection.transaction.results = new fixtures.results(this.connection)
```

Comment:
II think newer (less than 2 years old) versions of test-fixtures provide .results for us. Please confirm and remove this line.

Reply:
I replaced that line with:
  this.connection.init_transaction()